src/components/prod/Prod.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import SectionGuestLoves from '../sectionGuestLoves/SectionGuestLoves';
+
+import './App.css';
+
+function App() {
+  return (
+    <SectionGuestLoves/>
+  );
+}
+
+export default App;
```

Comment:
App.js file you cand store in components folder. The prod folder implies that there will be a non-prod version, is that so?

Reply:
corrected